
<!DOCTYPE html
  PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><head>
      <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
   <!--
This HTML was auto-generated from MATLAB code.
To make changes, update the MATLAB code and republish this document.
      --><title>Translatome cluster information page</title><meta name="generator" content="MATLAB 7.11"><link rel="schema.DC" href="http://purl.org/dc/elements/1.1/"><meta name="DC.date" content="2011-05-23"><meta name="DC.source" content="translatome_cluster_4.m"><style type="text/css">

body {
  background-color: white;
  margin:10px;
}

h1 {
  color: #990000; 
  font-size: x-large;
}

h2 {
  color: #990000;
  font-size: medium;
}

/* Make the text shrink to fit narrow windows, but not stretch too far in 
wide windows. */ 
p,h1,h2,div.content div {
  max-width: 600px;
  /* Hack for IE6 */
  width: auto !important; width: 600px;
}

pre.codeinput {
  background: #EEEEEE;
  padding: 10px;
}
@media print {
  pre.codeinput {word-wrap:break-word; width:100%;}
} 

span.keyword {color: #0000FF}
span.comment {color: #228B22}
span.string {color: #A020F0}
span.untermstring {color: #B20000}
span.syscmd {color: #B28C00}

pre.codeoutput {
  color: #666666;
  padding: 10px;
}

pre.error {
  color: red;
}

p.footer {
  text-align: right;
  font-size: xx-small;
  font-weight: lighter;
  font-style: italic;
  color: gray;
}

  </style></head><body><div class="content"><h1>Translatome cluster information page</h1><!--introduction--><!--/introduction--><h2>Contents</h2><div><ul><li><a href="#1">Cluster number</a></li><li><a href="#2">Genes in cluster</a></li><li><a href="#3">LeMoNe predicted transcriptome regulators</a></li><li><a href="#4">LeMoNe predicted translatome regulators</a></li><li><a href="#5">Translatome expression profiles</a></li><li><a href="#6">Transcriptome expression profiles</a></li><li><a href="#7">RAS figure</a></li><li><a href="#8">Enriched MIPS functional categories</a></li><li><a href="#9">Enriched MIPS protein complexes</a></li><li><a href="#10">Enriched cellular compartments</a></li><li><a href="#11">Enriched for known transcription factor targets</a></li><li><a href="#12">Enriched for known RNA binding protein targets</a></li><li><a href="#13">Regulated by mixed bifan</a></li><li><a href="#14">Regulated by posttranscriptional feedback loop</a></li><li><a href="#15">Regulated by transcriptional feedforward loop</a></li></ul></div><h2>Cluster number<a name="1"></a></h2><pre class="codeoutput">     4
</pre><h2>Genes in cluster<a name="2"></a></h2><pre class="codeoutput">NTH2
YBR138C
DUG2
LSB5
SYP1
YCR061W
RPT2
PHO2
YDL233W
NPP2
RAD4
ECM32
YER182W
PGD1
TIF4632
RAD54
CDC55
KEX1
SIP2
DBF2
YGR266W
YHL010C
OSH7
YHR159W
CST6
YIL080W
SDS3
BIR1
CDC16
SNF7
HOF1
CSM3
YMR086W
GLC8
HER1
</pre><h2>LeMoNe predicted transcriptome regulators<a name="3"></a></h2><pre class="codeoutput">EDS1
THI2
NRG1
GCN4
NDT80
ZAP1
RGT1
SIK1
SUT2
PUF2
</pre><h2>LeMoNe predicted translatome regulators<a name="4"></a></h2><pre class="codeoutput">EDS1
FAP7
ARO80
SPT2
HAC1
HMS2
ACE2
YLR278C
VTS1
DIG1
</pre><h2>Translatome expression profiles<a name="5"></a></h2><img vspace="5" hspace="5" src="translatome_cluster_4_01.png" alt=""> <h2>Transcriptome expression profiles<a name="6"></a></h2><img vspace="5" hspace="5" src="translatome_cluster_4_02.png" alt=""> <h2>RAS figure<a name="7"></a></h2><pre class="codeoutput">Class III
</pre><img vspace="5" hspace="5" src="translatome_cluster_4_03.png" alt=""> <h2>Enriched MIPS functional categories<a name="8"></a></h2><pre class="codeoutput">phosphate metabolism (P=0.0267)
regulation of C-compound and carbohydrate metabolism (P=0.0371)
DNA topology (P=0.00368)
DNA synthesis and replication (P=0.0118)
DNA repair (P=0.0128)
cell cycle checkpoints (checkpoints of morphogenesis, DNA-damage,-replication, mitotic phase and spindle) (P=0.000573)
cytokinesis (cell division) /septum formation and hydrolysis (P=0.00793)
chromosome segregation/division (P=0.00473)
transcriptional control (P=0.0102)
modification by phosphorylation, dephosphorylation, autophosphorylation (P=0.0221)
protein processing (proteolytic) (P=0.00568)
proteasomal degradation (ubiquitin/proteasomal pathway) (P=0.0378)
DNA damage response (P=0.0099)
actin cytoskeleton (P=0.0179)
</pre><h2>Enriched MIPS protein complexes<a name="9"></a></h2><pre class="codeoutput">None</pre><h2>Enriched cellular compartments<a name="10"></a></h2><pre class="codeoutput">neck (P=0.0035)
cell periphery (P=0.0322)
cytoplasm (P=0.0162)
cytoskeleton (P=0.00316)
nucleus (P=0.0135)
</pre><h2>Enriched for known transcription factor targets<a name="11"></a></h2><pre class="codeoutput">MBP1 regulates [PHO2, CDC55] (P=0.0448)
RME1 regulates [ECM32, CDC55] (P=0.0103)
NDT80 regulates [NTH2, DBF2] (P=0.00313)
WAR1 regulates [CST6, SDS3] (P=0.00568)
TYE7 regulates [ECM32, OSH7] (P=0.00793)
</pre><h2>Enriched for known RNA binding protein targets<a name="12"></a></h2><pre class="codeoutput">GBP2 regulates [RPT2, KEX1, SNF7] (P=0.0143)
</pre><h2>Regulated by mixed bifan<a name="13"></a></h2><pre class="codeoutput">No</pre><h2>Regulated by posttranscriptional feedback loop<a name="14"></a></h2><pre class="codeoutput">No</pre><h2>Regulated by transcriptional feedforward loop<a name="15"></a></h2><pre class="codeoutput">No</pre><p class="footer"><br>
      Published with MATLAB&reg; 7.11<br></p></div><!--
##### SOURCE BEGIN #####
%% Translatome cluster information page
%

%% Cluster number
%clnum = 24;
ix = find(ra_clusters_big(:,clnum));
disp(clnum)

%% Genes in cluster
gns = gene_names(ix);
for m=1:length(gns)
    fprintf('%s\n',gns{m});
end

%% LeMoNe predicted transcriptome regulators
trreg = gene_names(raclust_totalreg(:,clnum)>0);
for m=1:length(trreg)
    fprintf('%s\n',trreg{m});
end

%% LeMoNe predicted translatome regulators
tlreg = gene_names(raclust_rareg(:,clnum)>0);
for m=1:length(tlreg)
    fprintf('%s\n',tlreg{m});
end

%% Translatome expression profiles
plot(ra(ix,:)','b.-');
axis tight

%% Transcriptome expression profiles
plot(total(ix,:)','b.-');
axis tight

%% RAS figure
a1 = triu(ra_matrix(ix,ix),1);
a2 = tril(total_matrix(ix,ix),-1);
rs = rasnew(clnum); %full(sum(a2(:))/sum(a1(:)));
if rs<=0.1
    cls = 'Class III';
elseif rs<0.5
    cls = 'Class II';
else
    cls = 'Class I';
end
fprintf('%s\n',cls);
plotmatrix(a1+a2);
title(sprintf('RAS = %1.3f',rs));
colorbar

%% Enriched MIPS functional categories
pv = find(ramipspval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        fprintf('%s (P=%1.3g)\n',mips_names{m},ramipspval(clnum,m));
    end
end

%% Enriched MIPS protein complexes
pv = find(racomplexpval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        fprintf('%s (P=%1.3g)\n',complex_names{m},racomplexpval(clnum,m));
    end
end

%% Enriched cellular compartments
pv = find(ralocalpval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        fprintf('%s (P=%1.3g)\n',subcell_names{m},ralocalpval(clnum,m));
    end
end

%% Enriched for known transcription factor targets
tfs = find(sum(harb_med,2)>0);
pv = find(raharbpval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        tgt = intersect(find(harb_med(tfs(m),:)), ix');
        tgts = gene_names{tgt(1)};
        for g=2:length(tgt)
            tgts = strcat(tgts, sprintf(', %s',gene_names{tgt(g)}));
        end
        fprintf('%s regulates [%s] (P=%1.3g)\n', gene_names{tfs(m)}, tgts, raharbpval(clnum,m));
    end
end

%% Enriched for known RNA binding protein targets
rbps = find(sum(rbp,2)>0);
pv = find(rarbppval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        tgt = intersect(find(rbp(rbps(m),:)), ix');
        tgts = gene_names{tgt(1)};
        for g=2:length(tgt)
            tgts = strcat(tgts, sprintf(', %s',gene_names{tgt(g)}));
        end
        fprintf('%s regulates [%s] (P=%1.3g)\n', gene_names{rbps(m)}, tgts, rarbppval(clnum,m));
    end
end

%% Regulated by mixed bifan
if ismember(clnum,mixedbf)
    fprintf('Yes');
else
    fprintf('No');
end

%% Regulated by posttranscriptional feedback loop
if ismember(clnum,rbpfbl)
    fprintf('Yes');
else
    fprintf('No');
end

%% Regulated by transcriptional feedforward loop
if ismember(clnum,transffl)
    fprintf('Yes');
else
    fprintf('No');
end
##### SOURCE END #####
--></body></html>