
<!DOCTYPE html
  PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><head>
      <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
   <!--
This HTML was auto-generated from MATLAB code.
To make changes, update the MATLAB code and republish this document.
      --><title>Translatome cluster information page</title><meta name="generator" content="MATLAB 7.11"><link rel="schema.DC" href="http://purl.org/dc/elements/1.1/"><meta name="DC.date" content="2011-05-23"><meta name="DC.source" content="translatome_cluster_60.m"><style type="text/css">

body {
  background-color: white;
  margin:10px;
}

h1 {
  color: #990000; 
  font-size: x-large;
}

h2 {
  color: #990000;
  font-size: medium;
}

/* Make the text shrink to fit narrow windows, but not stretch too far in 
wide windows. */ 
p,h1,h2,div.content div {
  max-width: 600px;
  /* Hack for IE6 */
  width: auto !important; width: 600px;
}

pre.codeinput {
  background: #EEEEEE;
  padding: 10px;
}
@media print {
  pre.codeinput {word-wrap:break-word; width:100%;}
} 

span.keyword {color: #0000FF}
span.comment {color: #228B22}
span.string {color: #A020F0}
span.untermstring {color: #B20000}
span.syscmd {color: #B28C00}

pre.codeoutput {
  color: #666666;
  padding: 10px;
}

pre.error {
  color: red;
}

p.footer {
  text-align: right;
  font-size: xx-small;
  font-weight: lighter;
  font-style: italic;
  color: gray;
}

  </style></head><body><div class="content"><h1>Translatome cluster information page</h1><!--introduction--><!--/introduction--><h2>Contents</h2><div><ul><li><a href="#1">Cluster number</a></li><li><a href="#2">Genes in cluster</a></li><li><a href="#3">LeMoNe predicted transcriptome regulators</a></li><li><a href="#4">LeMoNe predicted translatome regulators</a></li><li><a href="#5">Translatome expression profiles</a></li><li><a href="#6">Transcriptome expression profiles</a></li><li><a href="#7">RAS figure</a></li><li><a href="#8">Enriched MIPS functional categories</a></li><li><a href="#9">Enriched MIPS protein complexes</a></li><li><a href="#10">Enriched cellular compartments</a></li><li><a href="#11">Enriched for known transcription factor targets</a></li><li><a href="#12">Enriched for known RNA binding protein targets</a></li><li><a href="#13">Regulated by mixed bifan</a></li><li><a href="#14">Regulated by posttranscriptional feedback loop</a></li><li><a href="#15">Regulated by transcriptional feedforward loop</a></li></ul></div><h2>Cluster number<a name="1"></a></h2><pre class="codeoutput">    60
</pre><h2>Genes in cluster<a name="2"></a></h2><pre class="codeoutput">ERP2
FUN12
RBG1
NCL1
PRE7
RPS8A
RPG1
MIS1
POL30
RPL31A
CDC53
CCT6
PMI40
SAH1
RPS8B
VTC2
RPL2A
LSG1
NOP7
TIF4631
RBG2
ELP2
ZPR1
GPI1
RPL8A
VMR1
DED81
YHR020W
VMA10
SSZ1
RRP3
DYS1
RPS4B
RPL2B
SFH5
ERG20
RPS22A
CCT5
LIA1
RPS5
RPS4A
MAK11
GPM1
FAS1
URA1
RPL15A
CBF5
TUB4
MAP1
RPS22B
RPL31B
RPS1B
URA5
RPL15B
RNA1
TIF11
MTQ1
LAT1
NCS2
GIM3
WRS1
HRP1
CDC33
YOR021C
RAT1
MCA1
ESA1
YOR246C
GCD1
ALA1
RPA43
PRT1
RPS12
EGD1
RPL5
BMS1
SUI3
TKL1
KAR3
</pre><h2>LeMoNe predicted transcriptome regulators<a name="3"></a></h2><pre class="codeoutput">SWI5
MET32
YAP6
CAD1
NPL3
CST6
MET28
STB2
RTG1
ARR1
</pre><h2>LeMoNe predicted translatome regulators<a name="4"></a></h2><pre class="codeoutput">HEK2
RTG3
PAB1
IME4
NSR1
MET28
CBF5
SIK1
BFR1
CUP9
</pre><h2>Translatome expression profiles<a name="5"></a></h2><img vspace="5" hspace="5" src="translatome_cluster_60_01.png" alt=""> <h2>Transcriptome expression profiles<a name="6"></a></h2><img vspace="5" hspace="5" src="translatome_cluster_60_02.png" alt=""> <h2>RAS figure<a name="7"></a></h2><pre class="codeoutput">Class II
</pre><img vspace="5" hspace="5" src="translatome_cluster_60_03.png" alt=""> <h2>Enriched MIPS functional categories<a name="8"></a></h2><pre class="codeoutput">amino acid metabolism (P=0.00178)
pyrimidine nucleotide/nucleoside/nucleobase metabolism (P=0.000669)
sugar, glucoside, polyol and carboxylate anabolism (P=0.0106)
sugar, glucoside, polyol and carboxylate catabolism (P=0.0216)
glycolysis and gluconeogenesis (P=0.0164)
mitotic cell cycle and cell cycle control (P=0.00198)
rRNA processing (P=0.0016)
tRNA modification (P=0.0186)
ribosome biogenesis (P=1.78e-05)
ribosomal proteins (P=1.59e-09)
translation initiation (P=1.47e-09)
aminoacyl-tRNA-synthetases (P=0.000141)
protein folding and stabilization (P=0.0336)
protein processing (proteolytic) (P=0.0497)
GTP binding (P=8.57e-06)
unfolded protein response (e.g. ER quality control) (P=0.00203)
cytoskeleton/structural proteins (P=0.00356)
</pre><h2>Enriched MIPS protein complexes<a name="9"></a></h2><pre class="codeoutput">Chaperonine containing T-complex TRiC (TCP RING Complex)  (P=0.000117)
rRNA splicing            (P=0.00364)
eIF4E/eIF4G/Pab1p complex (P=2.2e-06)
eIF3 (P=7.4e-05)
cytoplasmic ribosomal large subunit (P=9.08e-07)
cytoplasmic ribosomal small subunit (P=2.29e-09)
</pre><h2>Enriched cellular compartments<a name="10"></a></h2><pre class="codeoutput">cytoplasm (P=1.69e-08)
tubulin cytoskeleton (P=0.0106)
nucleolus (P=0.000307)
</pre><h2>Enriched for known transcription factor targets<a name="11"></a></h2><pre class="codeoutput">AFT1 regulates [LSG1, GCD1] (P=0.0404)
IME4 regulates [NOP7, HRP1] (P=0.0387)
RME1 regulates [RPL15A, RPS12, RPL5] (P=0.019)
ASK10 regulates [VTC2, RBG2, YOR021C, BMS1] (P=0.00216)
STP2 regulates [RPL31A, LAT1] (P=0.0497)
STB5 regulates [ESA1, TKL1] (P=0.0477)
IXR1 regulates [RBG2, RPS5, TIF11, NCS2] (P=0.00125)
RGT1 regulates [NCL1, RPL5] (P=0.00109)
ARG81 regulates [RPL31A, RPS1B, CDC33] (P=0.00467)
ARG80 regulates [RPL31A, NOP7, RPS12] (P=0.00973)
MOT3 regulates [MIS1, RPL5] (P=0.0249)
GCR2 regulates [RPL31A, RBG2, GPM1] (P=0.0199)
RAP1 regulates [RPS8A, RPS4B, RPS22A, RPS4A, GPM1, RPL31B] (P=0.0246)
STB1 regulates [VTC2, YOR246C] (P=0.0186)
HAL9 regulates [RPL31A, RBG2, CBF5, BMS1] (P=0.0064)
TYE7 regulates [ERP2, PMI40, GPM1] (P=0.0139)
MET31 regulates [RPL31A, CDC33] (P=0.037)
GCR1 regulates [GPM1, RPL5] (P=0.0277)
FHL1 regulates [RPS8A, RPL31A, RPL8A, RPS4B, RPS22A, RPS4A, RPL15A, RPL31B, RPS12] (P=1.35e-05)
</pre><h2>Enriched for known RNA binding protein targets<a name="12"></a></h2><pre class="codeoutput">PAB1 regulates [ERP2, PRE7, RPS8A, RPG1, POL30, RPL31A, RPS8B, TIF4631, RPL8A, YHR020W, VMA10, RPS4B, RPL2B, RPS22A, RPS5, RPS4A, GPM1, RPL15A, RPS22B, RPL31B, RPS1B, URA5, RPL15B, TIF11, MTQ1, CDC33, YOR021C, YOR246C, RPS12, EGD1] (P=0.000247)
PUF4 regulates [NCL1, POL30, DED81, CBF5, CDC33, ESA1] (P=0.02)
NSR1 regulates [RPL31A, RPS22B, RPL31B, TIF11, MTQ1, GIM3] (P=0.0142)
SCP160 regulates [FUN12, NCL1, RPG1, PMI40, VTC2, LSG1, NOP7, TIF4631, ELP2, YHR020W, SSZ1, MAK11, FAS1, TUB4, MAP1, LAT1, RAT1, MCA1, ALA1, PRT1, BMS1] (P=0.00709)
CBF5 regulates [RPS22A, RPS22B] (P=0.0477)
BFR1 regulates [FUN12, NCL1, RPG1, LSG1, NOP7, TIF4631, ELP2, RRP3, MAK11, FAS1, RAT1, BMS1] (P=0.00129)
NAB3 regulates [RPL2A, MAP1, RPS22B, RPL31B, URA5] (P=0.00498)
</pre><h2>Regulated by mixed bifan<a name="13"></a></h2><pre class="codeoutput">Yes</pre><h2>Regulated by posttranscriptional feedback loop<a name="14"></a></h2><pre class="codeoutput">Yes</pre><h2>Regulated by transcriptional feedforward loop<a name="15"></a></h2><pre class="codeoutput">Yes</pre><p class="footer"><br>
      Published with MATLAB&reg; 7.11<br></p></div><!--
##### SOURCE BEGIN #####
%% Translatome cluster information page
%

%% Cluster number
%clnum = 24;
ix = find(ra_clusters_big(:,clnum));
disp(clnum)

%% Genes in cluster
gns = gene_names(ix);
for m=1:length(gns)
    fprintf('%s\n',gns{m});
end

%% LeMoNe predicted transcriptome regulators
trreg = gene_names(raclust_totalreg(:,clnum)>0);
for m=1:length(trreg)
    fprintf('%s\n',trreg{m});
end

%% LeMoNe predicted translatome regulators
tlreg = gene_names(raclust_rareg(:,clnum)>0);
for m=1:length(tlreg)
    fprintf('%s\n',tlreg{m});
end

%% Translatome expression profiles
plot(ra(ix,:)','b.-');
axis tight

%% Transcriptome expression profiles
plot(total(ix,:)','b.-');
axis tight

%% RAS figure
a1 = triu(ra_matrix(ix,ix),1);
a2 = tril(total_matrix(ix,ix),-1);
rs = rasnew(clnum); %full(sum(a2(:))/sum(a1(:)));
if rs<=0.1
    cls = 'Class III';
elseif rs<0.5
    cls = 'Class II';
else
    cls = 'Class I';
end
fprintf('%s\n',cls);
plotmatrix(a1+a2);
title(sprintf('RAS = %1.3f',rs));
colorbar

%% Enriched MIPS functional categories
pv = find(ramipspval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        fprintf('%s (P=%1.3g)\n',mips_names{m},ramipspval(clnum,m));
    end
end

%% Enriched MIPS protein complexes
pv = find(racomplexpval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        fprintf('%s (P=%1.3g)\n',complex_names{m},racomplexpval(clnum,m));
    end
end

%% Enriched cellular compartments
pv = find(ralocalpval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        fprintf('%s (P=%1.3g)\n',subcell_names{m},ralocalpval(clnum,m));
    end
end

%% Enriched for known transcription factor targets
tfs = find(sum(harb_med,2)>0);
pv = find(raharbpval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        tgt = intersect(find(harb_med(tfs(m),:)), ix');
        tgts = gene_names{tgt(1)};
        for g=2:length(tgt)
            tgts = strcat(tgts, sprintf(', %s',gene_names{tgt(g)}));
        end
        fprintf('%s regulates [%s] (P=%1.3g)\n', gene_names{tfs(m)}, tgts, raharbpval(clnum,m));
    end
end

%% Enriched for known RNA binding protein targets
rbps = find(sum(rbp,2)>0);
pv = find(rarbppval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        tgt = intersect(find(rbp(rbps(m),:)), ix');
        tgts = gene_names{tgt(1)};
        for g=2:length(tgt)
            tgts = strcat(tgts, sprintf(', %s',gene_names{tgt(g)}));
        end
        fprintf('%s regulates [%s] (P=%1.3g)\n', gene_names{rbps(m)}, tgts, rarbppval(clnum,m));
    end
end

%% Regulated by mixed bifan
if ismember(clnum,mixedbf)
    fprintf('Yes');
else
    fprintf('No');
end

%% Regulated by posttranscriptional feedback loop
if ismember(clnum,rbpfbl)
    fprintf('Yes');
else
    fprintf('No');
end

%% Regulated by transcriptional feedforward loop
if ismember(clnum,transffl)
    fprintf('Yes');
else
    fprintf('No');
end
##### SOURCE END #####
--></body></html>