
<!DOCTYPE html
  PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><head>
      <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
   <!--
This HTML was auto-generated from MATLAB code.
To make changes, update the MATLAB code and republish this document.
      --><title>Transcriptome cluster information page</title><meta name="generator" content="MATLAB 7.11"><link rel="schema.DC" href="http://purl.org/dc/elements/1.1/"><meta name="DC.date" content="2011-06-08"><meta name="DC.source" content="transcriptome_cluster_5.m"><style type="text/css">

body {
  background-color: white;
  margin:10px;
}

h1 {
  color: #990000; 
  font-size: x-large;
}

h2 {
  color: #990000;
  font-size: medium;
}

/* Make the text shrink to fit narrow windows, but not stretch too far in 
wide windows. */ 
p,h1,h2,div.content div {
  max-width: 600px;
  /* Hack for IE6 */
  width: auto !important; width: 600px;
}

pre.codeinput {
  background: #EEEEEE;
  padding: 10px;
}
@media print {
  pre.codeinput {word-wrap:break-word; width:100%;}
} 

span.keyword {color: #0000FF}
span.comment {color: #228B22}
span.string {color: #A020F0}
span.untermstring {color: #B20000}
span.syscmd {color: #B28C00}

pre.codeoutput {
  color: #666666;
  padding: 10px;
}

pre.error {
  color: red;
}

p.footer {
  text-align: right;
  font-size: xx-small;
  font-weight: lighter;
  font-style: italic;
  color: gray;
}

  </style></head><body><div class="content"><h1>Transcriptome cluster information page</h1><!--introduction--><!--/introduction--><h2>Contents</h2><div><ul><li><a href="#1">Cluster number</a></li><li><a href="#2">Genes in cluster</a></li><li><a href="#3">LeMoNe predicted transcriptome regulators</a></li><li><a href="#4">Transcriptome expression profiles</a></li><li><a href="#5">Enriched MIPS functional categories</a></li><li><a href="#6">Enriched MIPS protein complexes</a></li><li><a href="#7">Enriched cellular compartments</a></li><li><a href="#8">Enriched for known transcription factor targets</a></li><li><a href="#9">Enriched for known RNA binding protein targets</a></li></ul></div><h2>Cluster number<a name="1"></a></h2><pre class="codeoutput">     5
</pre><h2>Genes in cluster<a name="2"></a></h2><pre class="codeoutput">URA7
TAT1
MAK5
PSA1
RPS29B
GDA1
NUG1
PRP43
HXK2
PPT1
ENP2
SDA1
PRS3
RPS27B
SSZ1
IMD2
TIF2
RPL43B
TMA19
TIF1
RPS0B
DPH5
SIK1
SUR4
ASC1
HAS1
RPL16B
SSB2
RPL18B
DBP6
GLN4
RPL20B
ELP3
SSE1
RPO26
</pre><h2>LeMoNe predicted transcriptome regulators<a name="3"></a></h2><pre class="codeoutput">ARO80
DOT6
NSR1
ZAP1
SPT10
CBF5
SIK1
NAB6
SOK2
TYE7
</pre><h2>Transcriptome expression profiles<a name="4"></a></h2><img vspace="5" hspace="5" src="transcriptome_cluster_5_01.png" alt=""> <h2>Enriched MIPS functional categories<a name="5"></a></h2><pre class="codeoutput">purine nucleotide/nucleoside/nucleobase anabolism (P=0.000599)
pyrimidine nucleotide/nucleoside/nucleobase anabolism (P=6.34e-05)
phosphate metabolism (P=0.0267)
rRNA processing (P=4.29e-05)
splicing (P=0.044)
PROTEIN SYNTHESIS (P=0.000431)
ribosome biogenesis (P=2.99e-05)
ribosomal proteins (P=8e-07)
translation initiation (P=0.00155)
modification with sugar residues (e.g. glycosylation, deglycosylation) (P=0.000117)
ATP binding (P=0.00463)
unfolded protein response (e.g. ER quality control) (P=0.000641)
</pre><h2>Enriched MIPS protein complexes<a name="6"></a></h2><pre class="codeoutput">mRNA splicing (P=0.00123)
eIF4A (P=1.94e-11)
cytoplasmic ribosomal large subunit (P=9.36e-05)
cytoplasmic ribosomal small subunit (P=1.69e-05)
</pre><h2>Enriched cellular compartments<a name="7"></a></h2><pre class="codeoutput">cytoplasm (P=0.000715)
nucleolus (P=0.00102)
</pre><h2>Enriched for known transcription factor targets<a name="8"></a></h2><pre class="codeoutput">SMP1 regulates [RPS29B, RPS27B] (P=0.0281)
PDR1 regulates [RPS27B, RPL16B] (P=0.0239)
CST6 regulates [IMD2, TIF1] (P=0.0062)
YAP5 regulates [RPS27B, RPL16B] (P=0.02)
DAL81 regulates [TAT1, HXK2] (P=0.00856)
HIR3 regulates [MAK5, RPS0B] (P=0.00956)
PHD1 regulates [TIF2, RPL43B] (P=0.0233)
GAT3 regulates [RPS27B, RPL16B] (P=0.0147)
SWI6 regulates [PSA1, RPL18B] (P=0.044)
YOX1 regulates [MAK5, RPS0B, SSE1] (P=0.00238)
DAT1 regulates [MAK5, IMD2] (P=0.00279)
THO2 regulates [MAK5, SSE1] (P=0.0121)
RAP1 regulates [RPS29B, RPS27B, ASC1, RPL16B, RPL20B] (P=0.00147)
HAL9 regulates [MAK5, IMD2, SSE1] (P=0.00174)
MET31 regulates [RPS29B, RPS0B] (P=0.00408)
AFT2 regulates [TIF1, SUR4] (P=0.00349)
FHL1 regulates [RPS29B, RPS27B, ASC1, RPL16B, RPL20B] (P=0.00014)
</pre><h2>Enriched for known RNA binding protein targets<a name="9"></a></h2><pre class="codeoutput">PUF4 regulates [URA7, GDA1, ASC1, RPL18B, RPL20B] (P=0.00119)
SCP160 regulates [URA7, TAT1, MAK5, NUG1, PRP43, HXK2, ENP2, SDA1, SSZ1, IMD2, TIF2, TIF1, SUR4, HAS1, DBP6, GLN4, ELP3, SSE1] (P=3.13e-07)
MSL5 regulates [RPS29B, RPL18B] (P=0.00451)
NAB6 regulates [RPS29B, TMA19] (P=0.00313)
PUB1 regulates [URA7, TAT1, MAK5, RPS29B, GDA1, HXK2, PPT1, PRS3, RPS27B, TIF2, TMA19, TIF1, SUR4, HAS1, SSB2, RPL18B, GLN4, RPO26] (P=0.000936)
BFR1 regulates [MAK5, PRP43, ENP2, TIF2, TIF1, HAS1, ELP3, SSE1] (P=0.00025)
CBC2 regulates [RPS29B, RPL43B, RPL18B] (P=0.00709)
</pre><p class="footer"><br>
      Published with MATLAB&reg; 7.11<br></p></div><!--
##### SOURCE BEGIN #####
%% Transcriptome cluster information page
%

%% Cluster number
%clnum = 1;
ix = find(total_clusters_big(:,clnum));
disp(clnum)

%% Genes in cluster
gns = gene_names(ix);
for m=1:length(gns)
    fprintf('%s\n',gns{m});
end

%% LeMoNe predicted transcriptome regulators
trreg = gene_names(totalclust_totalreg(:,clnum)>0);
for m=1:length(trreg)
    fprintf('%s\n',trreg{m});
end

%% Transcriptome expression profiles
plot(total(ix,:)','b.-');
axis tight

%% Enriched MIPS functional categories
pv = find(totalmipspval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        fprintf('%s (P=%1.3g)\n',mips_names{m},totalmipspval(clnum,m));
    end
end

%% Enriched MIPS protein complexes
pv = find(totalcomplexpval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        fprintf('%s (P=%1.3g)\n',complex_names{m},totalcomplexpval(clnum,m));
    end
end

%% Enriched cellular compartments
pv = find(totallocalpval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        fprintf('%s (P=%1.3g)\n',subcell_names{m},totallocalpval(clnum,m));
    end
end

%% Enriched for known transcription factor targets
tfs = find(sum(harb_med,2)>0);
pv = find(totalharbpval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        tgt = intersect(find(harb_med(tfs(m),:)), ix');
        tgts = gene_names{tgt(1)};
        for g=2:length(tgt)
            tgts = strcat(tgts, sprintf(', %s',gene_names{tgt(g)}));
        end
        fprintf('%s regulates [%s] (P=%1.3g)\n', gene_names{tfs(m)}, tgts, totalharbpval(clnum,m));
    end
end

%% Enriched for known RNA binding protein targets
rbps = find(sum(rbp,2)>0);
pv = find(totalrbppval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        tgt = intersect(find(rbp(rbps(m),:)), ix');
        tgts = gene_names{tgt(1)};
        for g=2:length(tgt)
            tgts = strcat(tgts, sprintf(', %s',gene_names{tgt(g)}));
        end
        fprintf('%s regulates [%s] (P=%1.3g)\n', gene_names{rbps(m)}, tgts, totalrbppval(clnum,m));
    end
end
##### SOURCE END #####
--></body></html>