
<!DOCTYPE html
  PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><head>
      <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
   <!--
This HTML was auto-generated from MATLAB code.
To make changes, update the MATLAB code and republish this document.
      --><title>Translatome cluster information page</title><meta name="generator" content="MATLAB 7.11"><link rel="schema.DC" href="http://purl.org/dc/elements/1.1/"><meta name="DC.date" content="2011-05-23"><meta name="DC.source" content="translatome_cluster_41.m"><style type="text/css">

body {
  background-color: white;
  margin:10px;
}

h1 {
  color: #990000; 
  font-size: x-large;
}

h2 {
  color: #990000;
  font-size: medium;
}

/* Make the text shrink to fit narrow windows, but not stretch too far in 
wide windows. */ 
p,h1,h2,div.content div {
  max-width: 600px;
  /* Hack for IE6 */
  width: auto !important; width: 600px;
}

pre.codeinput {
  background: #EEEEEE;
  padding: 10px;
}
@media print {
  pre.codeinput {word-wrap:break-word; width:100%;}
} 

span.keyword {color: #0000FF}
span.comment {color: #228B22}
span.string {color: #A020F0}
span.untermstring {color: #B20000}
span.syscmd {color: #B28C00}

pre.codeoutput {
  color: #666666;
  padding: 10px;
}

pre.error {
  color: red;
}

p.footer {
  text-align: right;
  font-size: xx-small;
  font-weight: lighter;
  font-style: italic;
  color: gray;
}

  </style></head><body><div class="content"><h1>Translatome cluster information page</h1><!--introduction--><!--/introduction--><h2>Contents</h2><div><ul><li><a href="#1">Cluster number</a></li><li><a href="#2">Genes in cluster</a></li><li><a href="#3">LeMoNe predicted transcriptome regulators</a></li><li><a href="#4">LeMoNe predicted translatome regulators</a></li><li><a href="#5">Translatome expression profiles</a></li><li><a href="#6">Transcriptome expression profiles</a></li><li><a href="#7">RAS figure</a></li><li><a href="#8">Enriched MIPS functional categories</a></li><li><a href="#9">Enriched MIPS protein complexes</a></li><li><a href="#10">Enriched cellular compartments</a></li><li><a href="#11">Enriched for known transcription factor targets</a></li><li><a href="#12">Enriched for known RNA binding protein targets</a></li><li><a href="#13">Regulated by mixed bifan</a></li><li><a href="#14">Regulated by posttranscriptional feedback loop</a></li><li><a href="#15">Regulated by transcriptional feedforward loop</a></li></ul></div><h2>Cluster number<a name="1"></a></h2><pre class="codeoutput">    41
</pre><h2>Genes in cluster<a name="2"></a></h2><pre class="codeoutput">ADH5
MET8
GGC1
MET32
IES5
TMT1
LEU1
STR3
MUP1
SER33
YIL152W
YIL165C
YAP5
STR2
MET14
MET1
MTD1
SUL2
ADI1
ECM40
SNZ1
MTG1
ADE12
MET2
TAT2
MET22
MCH4
ARG8
ORT1
ICY2
ICL2
MET16
</pre><h2>LeMoNe predicted transcriptome regulators<a name="3"></a></h2><pre class="codeoutput">MET32
YAP6
CAD1
MET28
STB6
ECM22
MOT3
MSS11
KRE33
HMS1
</pre><h2>LeMoNe predicted translatome regulators<a name="4"></a></h2><pre class="codeoutput">CAD1
HAC1
OTU1
MET28
YAP5
HRB1
HMS1
BFR1
GCR1
CUP9
</pre><h2>Translatome expression profiles<a name="5"></a></h2><img vspace="5" hspace="5" src="translatome_cluster_41_01.png" alt=""> <h2>Transcriptome expression profiles<a name="6"></a></h2><img vspace="5" hspace="5" src="translatome_cluster_41_02.png" alt=""> <h2>RAS figure<a name="7"></a></h2><pre class="codeoutput">Class II
</pre><img vspace="5" hspace="5" src="translatome_cluster_41_03.png" alt=""> <h2>Enriched MIPS functional categories<a name="8"></a></h2><pre class="codeoutput">amino acid metabolism (P=5.32e-05)
biosynthesis of arginine (P=4.61e-07)
metabolism of methionine (P=5.96e-08)
metabolism of cysteine (P=1.13e-05)
nitrogen, sulfur and selenium metabolism (P=0.003)
sulfate assimilation (P=4.19e-12)
regulation of nitrogen, sulfur and selenium metabolism (P=0.000198)
purine nucleotide/nucleoside/nucleobase anabolism (P=0.000458)
biosynthesis of vitamins, cofactors, and prosthetic groups (P=0.0202)
metabolism of nonprotein amino acids (P=1.61e-05)
metabolism of porphyrins (P=8.73e-07)
general transcription activities (P=0.0418)
NAD/NADP binding (P=3.48e-05)
amino acid/amino acid derivatives transport (P=8.52e-05)
mitochondrial transport (P=0.017)
cellular import (P=0.0122)
TRANSPOSABLE ELEMENTS, VIRAL AND PLASMID PROTEINS (P=0.00169)
</pre><h2>Enriched MIPS protein complexes<a name="9"></a></h2><pre class="codeoutput">None</pre><h2>Enriched cellular compartments<a name="10"></a></h2><pre class="codeoutput">cytoplasm (P=0.0217)
mitochondrial inner membrane (P=0.0404)
mitochondrial matrix (P=0.00592)
</pre><h2>Enriched for known transcription factor targets<a name="11"></a></h2><pre class="codeoutput">OAF1 regulates [MET8, ECM40] (P=0.0122)
MET32 regulates [MET8, STR3, MET22, MET16] (P=0.000156)
YAP6 regulates [ADH5, YAP5] (P=0.0222)
GCN4 regulates [GGC1, STR3, ECM40, MET22, MCH4, ORT1] (P=1.91e-07)
HAP2 regulates [ADH5, ARG8] (P=0.00315)
YAP1 regulates [ADH5, MCH4] (P=0.0125)
MET31 regulates [STR3, MET16] (P=0.00315)
</pre><h2>Enriched for known RNA binding protein targets<a name="12"></a></h2><pre class="codeoutput">PAB1 regulates [ADH5, IES5, SER33, YIL152W, YIL165C, MET14, ADI1, ARG8, ORT1, ICY2, ICL2, MET16] (P=0.0112)
SIK1 regulates [MUP1, MCH4] (P=0.0183)
VTS1 regulates [SER33, TAT2, MCH4] (P=0.00377)
</pre><h2>Regulated by mixed bifan<a name="13"></a></h2><pre class="codeoutput">Yes</pre><h2>Regulated by posttranscriptional feedback loop<a name="14"></a></h2><pre class="codeoutput">No</pre><h2>Regulated by transcriptional feedforward loop<a name="15"></a></h2><pre class="codeoutput">Yes</pre><p class="footer"><br>
      Published with MATLAB&reg; 7.11<br></p></div><!--
##### SOURCE BEGIN #####
%% Translatome cluster information page
%

%% Cluster number
%clnum = 24;
ix = find(ra_clusters_big(:,clnum));
disp(clnum)

%% Genes in cluster
gns = gene_names(ix);
for m=1:length(gns)
    fprintf('%s\n',gns{m});
end

%% LeMoNe predicted transcriptome regulators
trreg = gene_names(raclust_totalreg(:,clnum)>0);
for m=1:length(trreg)
    fprintf('%s\n',trreg{m});
end

%% LeMoNe predicted translatome regulators
tlreg = gene_names(raclust_rareg(:,clnum)>0);
for m=1:length(tlreg)
    fprintf('%s\n',tlreg{m});
end

%% Translatome expression profiles
plot(ra(ix,:)','b.-');
axis tight

%% Transcriptome expression profiles
plot(total(ix,:)','b.-');
axis tight

%% RAS figure
a1 = triu(ra_matrix(ix,ix),1);
a2 = tril(total_matrix(ix,ix),-1);
rs = rasnew(clnum); %full(sum(a2(:))/sum(a1(:)));
if rs<=0.1
    cls = 'Class III';
elseif rs<0.5
    cls = 'Class II';
else
    cls = 'Class I';
end
fprintf('%s\n',cls);
plotmatrix(a1+a2);
title(sprintf('RAS = %1.3f',rs));
colorbar

%% Enriched MIPS functional categories
pv = find(ramipspval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        fprintf('%s (P=%1.3g)\n',mips_names{m},ramipspval(clnum,m));
    end
end

%% Enriched MIPS protein complexes
pv = find(racomplexpval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        fprintf('%s (P=%1.3g)\n',complex_names{m},racomplexpval(clnum,m));
    end
end

%% Enriched cellular compartments
pv = find(ralocalpval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        fprintf('%s (P=%1.3g)\n',subcell_names{m},ralocalpval(clnum,m));
    end
end

%% Enriched for known transcription factor targets
tfs = find(sum(harb_med,2)>0);
pv = find(raharbpval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        tgt = intersect(find(harb_med(tfs(m),:)), ix');
        tgts = gene_names{tgt(1)};
        for g=2:length(tgt)
            tgts = strcat(tgts, sprintf(', %s',gene_names{tgt(g)}));
        end
        fprintf('%s regulates [%s] (P=%1.3g)\n', gene_names{tfs(m)}, tgts, raharbpval(clnum,m));
    end
end

%% Enriched for known RNA binding protein targets
rbps = find(sum(rbp,2)>0);
pv = find(rarbppval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        tgt = intersect(find(rbp(rbps(m),:)), ix');
        tgts = gene_names{tgt(1)};
        for g=2:length(tgt)
            tgts = strcat(tgts, sprintf(', %s',gene_names{tgt(g)}));
        end
        fprintf('%s regulates [%s] (P=%1.3g)\n', gene_names{rbps(m)}, tgts, rarbppval(clnum,m));
    end
end

%% Regulated by mixed bifan
if ismember(clnum,mixedbf)
    fprintf('Yes');
else
    fprintf('No');
end

%% Regulated by posttranscriptional feedback loop
if ismember(clnum,rbpfbl)
    fprintf('Yes');
else
    fprintf('No');
end

%% Regulated by transcriptional feedforward loop
if ismember(clnum,transffl)
    fprintf('Yes');
else
    fprintf('No');
end
##### SOURCE END #####
--></body></html>