
<!DOCTYPE html
  PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><head>
      <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
   <!--
This HTML was auto-generated from MATLAB code.
To make changes, update the MATLAB code and republish this document.
      --><title>Translatome cluster information page</title><meta name="generator" content="MATLAB 7.11"><link rel="schema.DC" href="http://purl.org/dc/elements/1.1/"><meta name="DC.date" content="2011-05-23"><meta name="DC.source" content="translatome_cluster_34.m"><style type="text/css">

body {
  background-color: white;
  margin:10px;
}

h1 {
  color: #990000; 
  font-size: x-large;
}

h2 {
  color: #990000;
  font-size: medium;
}

/* Make the text shrink to fit narrow windows, but not stretch too far in 
wide windows. */ 
p,h1,h2,div.content div {
  max-width: 600px;
  /* Hack for IE6 */
  width: auto !important; width: 600px;
}

pre.codeinput {
  background: #EEEEEE;
  padding: 10px;
}
@media print {
  pre.codeinput {word-wrap:break-word; width:100%;}
} 

span.keyword {color: #0000FF}
span.comment {color: #228B22}
span.string {color: #A020F0}
span.untermstring {color: #B20000}
span.syscmd {color: #B28C00}

pre.codeoutput {
  color: #666666;
  padding: 10px;
}

pre.error {
  color: red;
}

p.footer {
  text-align: right;
  font-size: xx-small;
  font-weight: lighter;
  font-style: italic;
  color: gray;
}

  </style></head><body><div class="content"><h1>Translatome cluster information page</h1><!--introduction--><!--/introduction--><h2>Contents</h2><div><ul><li><a href="#1">Cluster number</a></li><li><a href="#2">Genes in cluster</a></li><li><a href="#3">LeMoNe predicted transcriptome regulators</a></li><li><a href="#4">LeMoNe predicted translatome regulators</a></li><li><a href="#5">Translatome expression profiles</a></li><li><a href="#6">Transcriptome expression profiles</a></li><li><a href="#7">RAS figure</a></li><li><a href="#8">Enriched MIPS functional categories</a></li><li><a href="#9">Enriched MIPS protein complexes</a></li><li><a href="#10">Enriched cellular compartments</a></li><li><a href="#11">Enriched for known transcription factor targets</a></li><li><a href="#12">Enriched for known RNA binding protein targets</a></li><li><a href="#13">Regulated by mixed bifan</a></li><li><a href="#14">Regulated by posttranscriptional feedback loop</a></li><li><a href="#15">Regulated by transcriptional feedforward loop</a></li></ul></div><h2>Cluster number<a name="1"></a></h2><pre class="codeoutput">    34
</pre><h2>Genes in cluster<a name="2"></a></h2><pre class="codeoutput">PET9
COR1
ATP1
VMA2
PDB1
ALG7
SSH1
PMP1
PMT1
VCX1
PST1
WBP1
RPN3
RSP5
PDA1
ERG4
STT3
HNM1
GUP1
VRG4
PFK1
TNA1
BGL2
GPI16
OST1
MNN5
YJR015W
SSC1
GFA1
ACS2
TCB3
POR1
YNR021W
KES1
QCR2
</pre><h2>LeMoNe predicted transcriptome regulators<a name="3"></a></h2><pre class="codeoutput">RTG3
KSS1
RIM101
SCP160
PPR1
MSL5
SWI6
SOK2
RGM1
HAP5
</pre><h2>LeMoNe predicted translatome regulators<a name="4"></a></h2><pre class="codeoutput">HAP3
TBS1
SMP1
GBP2
SRD1
MBP1
NRG1
HOG1
MSN1
YPR196W
</pre><h2>Translatome expression profiles<a name="5"></a></h2><img vspace="5" hspace="5" src="translatome_cluster_34_01.png" alt=""> <h2>Transcriptome expression profiles<a name="6"></a></h2><img vspace="5" hspace="5" src="translatome_cluster_34_02.png" alt=""> <h2>RAS figure<a name="7"></a></h2><pre class="codeoutput">Class III
</pre><img vspace="5" hspace="5" src="translatome_cluster_34_03.png" alt=""> <h2>Enriched MIPS functional categories<a name="8"></a></h2><pre class="codeoutput">C-compound and carbohydrate metabolism (P=0.0395)
polysaccharide metabolism (P=0.00105)
peptidoglycan anabolism (P=2.19e-06)
tetracyclic and pentacyclic triterpenes (cholesterin, steroids and hopanoids) metabolism (P=0.00114)
glycolysis and gluconeogenesis (P=8.43e-05)
pyruvate dehydrogenase complex (P=1.8e-06)
electron transport and membrane-associated energy conservation (P=0.000268)
aerobic respiration (P=6.46e-05)
N-directed glycosylation, deglycosylation (P=1.31e-07)
proteasomal degradation (ubiquitin/proteasomal pathway) (P=0.0378)
enzymatic activity regulation / enzyme regulator (P=0.0103)
cation transport (H+, Na+, K+, Ca2+ , NH4+, etc.) (P=3.74e-05)
protein transport (P=0.0481)
electron transport (P=0.011)
nucleotide/nucleoside/nucleobase transport (P=0.000194)
transport ATPases (P=0.000215)
mitochondrial transport (P=0.000292)
vesicular transport (Golgi network, etc.) (P=0.00824)
vacuolar/lysosomal transport (P=0.0115)
homeostasis of protons (P=5.8e-06)
cell wall (P=0.0342)
</pre><h2>Enriched MIPS protein complexes<a name="9"></a></h2><pre class="codeoutput">Pyruvate dehydrogenase (P=1.8e-06)
Cytochrome bc1 complex (Ubiquinol-cytochrome c reductase complex, complex III) (P=2.12e-05)
Oligosaccharyltransferase (P=1.2e-07)
</pre><h2>Enriched cellular compartments<a name="10"></a></h2><pre class="codeoutput">cell periphery (P=0.0322)
plasma membrane (P=0.0213)
integral membrane (P=0.0149)
ER (P=7.32e-11)
ER membrane (P=0.000794)
golgi (P=0.0349)
golgi membrane (P=0.016)
nuclear envelope (P=3.14e-05)
mitochondria (P=0.00298)
mitochondrial inner membrane (P=0.00135)
vacuole (P=0.0373)
vacuolar membrane (P=0.0175)
</pre><h2>Enriched for known transcription factor targets<a name="11"></a></h2><pre class="codeoutput">OAF1 regulates [VMA2, SSC1, TCB3] (P=0.00181)
REB1 regulates [OST1, GFA1] (P=0.0464)
SMP1 regulates [HNM1, BGL2] (P=0.0281)
MET32 regulates [VMA2, TCB3] (P=0.0195)
SUT1 regulates [PET9, ACS2] (P=0.0288)
STP2 regulates [YJR015W, YNR021W] (P=0.00568)
HAP4 regulates [PET9, COR1, POR1] (P=0.00141)
ABF1 regulates [SSH1, PFK1, BGL2, SSC1] (P=0.0265)
PPR1 regulates [VMA2, RPN3] (P=0.00313)
SWI6 regulates [PET9, VCX1] (P=0.044)
PIP2 regulates [VMA2, SSC1, TCB3] (P=0.00051)
CUP9 regulates [PET9, HNM1, YJR015W] (P=0.000715)
ROX1 regulates [PET9, ATP1] (P=0.0416)
</pre><h2>Enriched for known RNA binding protein targets<a name="12"></a></h2><pre class="codeoutput">TDH3 regulates [SSH1, RPN3, GPI16, OST1] (P=4.02e-05)
SCP160 regulates [COR1, ALG7, SSH1, PMT1, PST1, RPN3, RSP5, ERG4, STT3, HNM1, GUP1, VRG4, PFK1, GPI16, OST1, MNN5, YJR015W, GFA1, ACS2, TCB3, YNR021W] (P=7.92e-10)
YLL032C regulates [PMP1, STT3, GPI16] (P=0.000232)
SIK1 regulates [PMT1, GPI16] (P=0.0233)
PUB1 regulates [PET9, PDB1, SSH1, PMP1, VCX1, RSP5, ERG4, HNM1, GUP1, VRG4, PFK1, TNA1, BGL2, OST1, YJR015W, SSC1, POR1, KES1, QCR2] (P=0.000278)
BFR1 regulates [COR1, ALG7, RPN3, ERG4, HNM1, VRG4, GFA1, ACS2] (P=0.00025)
YPL184C regulates [SSH1, PMP1, VCX1, VRG4, TNA1, BGL2, YJR015W] (P=7.6e-05)
</pre><h2>Regulated by mixed bifan<a name="13"></a></h2><pre class="codeoutput">Yes</pre><h2>Regulated by posttranscriptional feedback loop<a name="14"></a></h2><pre class="codeoutput">Yes</pre><h2>Regulated by transcriptional feedforward loop<a name="15"></a></h2><pre class="codeoutput">No</pre><p class="footer"><br>
      Published with MATLAB&reg; 7.11<br></p></div><!--
##### SOURCE BEGIN #####
%% Translatome cluster information page
%

%% Cluster number
%clnum = 24;
ix = find(ra_clusters_big(:,clnum));
disp(clnum)

%% Genes in cluster
gns = gene_names(ix);
for m=1:length(gns)
    fprintf('%s\n',gns{m});
end

%% LeMoNe predicted transcriptome regulators
trreg = gene_names(raclust_totalreg(:,clnum)>0);
for m=1:length(trreg)
    fprintf('%s\n',trreg{m});
end

%% LeMoNe predicted translatome regulators
tlreg = gene_names(raclust_rareg(:,clnum)>0);
for m=1:length(tlreg)
    fprintf('%s\n',tlreg{m});
end

%% Translatome expression profiles
plot(ra(ix,:)','b.-');
axis tight

%% Transcriptome expression profiles
plot(total(ix,:)','b.-');
axis tight

%% RAS figure
a1 = triu(ra_matrix(ix,ix),1);
a2 = tril(total_matrix(ix,ix),-1);
rs = rasnew(clnum); %full(sum(a2(:))/sum(a1(:)));
if rs<=0.1
    cls = 'Class III';
elseif rs<0.5
    cls = 'Class II';
else
    cls = 'Class I';
end
fprintf('%s\n',cls);
plotmatrix(a1+a2);
title(sprintf('RAS = %1.3f',rs));
colorbar

%% Enriched MIPS functional categories
pv = find(ramipspval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        fprintf('%s (P=%1.3g)\n',mips_names{m},ramipspval(clnum,m));
    end
end

%% Enriched MIPS protein complexes
pv = find(racomplexpval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        fprintf('%s (P=%1.3g)\n',complex_names{m},racomplexpval(clnum,m));
    end
end

%% Enriched cellular compartments
pv = find(ralocalpval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        fprintf('%s (P=%1.3g)\n',subcell_names{m},ralocalpval(clnum,m));
    end
end

%% Enriched for known transcription factor targets
tfs = find(sum(harb_med,2)>0);
pv = find(raharbpval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        tgt = intersect(find(harb_med(tfs(m),:)), ix');
        tgts = gene_names{tgt(1)};
        for g=2:length(tgt)
            tgts = strcat(tgts, sprintf(', %s',gene_names{tgt(g)}));
        end
        fprintf('%s regulates [%s] (P=%1.3g)\n', gene_names{tfs(m)}, tgts, raharbpval(clnum,m));
    end
end

%% Enriched for known RNA binding protein targets
rbps = find(sum(rbp,2)>0);
pv = find(rarbppval(clnum,:)<0.05);
if isempty(pv)
    fprintf('None');
else
    for m=pv
        tgt = intersect(find(rbp(rbps(m),:)), ix');
        tgts = gene_names{tgt(1)};
        for g=2:length(tgt)
            tgts = strcat(tgts, sprintf(', %s',gene_names{tgt(g)}));
        end
        fprintf('%s regulates [%s] (P=%1.3g)\n', gene_names{rbps(m)}, tgts, rarbppval(clnum,m));
    end
end

%% Regulated by mixed bifan
if ismember(clnum,mixedbf)
    fprintf('Yes');
else
    fprintf('No');
end

%% Regulated by posttranscriptional feedback loop
if ismember(clnum,rbpfbl)
    fprintf('Yes');
else
    fprintf('No');
end

%% Regulated by transcriptional feedforward loop
if ismember(clnum,transffl)
    fprintf('Yes');
else
    fprintf('No');
end
##### SOURCE END #####
--></body></html>